### PR TITLE
Moving signupEmail to service and adding to proxy-pki user creation

### PR DESCRIPTION
**Pull request by matthewgaulin**
This is a follow on to: https://github.com/Asymmetrik/mean2-starter/pull/77

Forgot to add the email for a user created in proxy-pki mode.  Also moved the new user email function to the service layer to be accessed by the proxy-pki strategy

### CODE REVIEW

**Review by brianghig — APPROVED** [review]

File: src/server/app/admin/services/users.authentication.server.service.js [hunk 1]
```diff
@@ -44,6 +46,46 @@ module.exports.initializeNewUser = function(user) {
...
 
+// Send email alert about new account request
+module.exports.signupEmail = function(user, req) {
+	let defer = q.defer();
+
+	req.app.render('../../admin/templates/user-signup-alert-email', {
+		name: user.name,
+		username: user.username,
+		appName: config.app.title,
+		url: `${config.app.baseUrl}/#/admin/users`
```
Comment (ymao1) [inline src/server/app/admin/services/users.authentication.server.service.js]: this hashmark will need to be removed when your other PR gets merged.

Comment (matthewgaulin) [reply src/server/app/admin/services/users.authentication.server.service.js]: Good call.  Depending on which one gets merged first I will make the change in the other.

Comment (rbutterfield) [reply src/server/app/admin/services/users.authentication.server.service.js]: can this be addressed now?

Comment (matthewgaulin) [reply src/server/app/admin/services/users.authentication.server.service.js]: Just addressed this

**Review by rbutterfield — APPROVED** [review]

**Pull request merged**